### PR TITLE
Openspace fix

### PR DESCRIPTION
**Pull request by Camper12345**
* Added can_fall and can_prevent_fall procs
* Added damage by falling objects
* Added possiblity to add fall sound

### CODE REVIEW

**Comment by Camper12345** [comment]
#614

File: code/__defines/_compile_options.dm [hunk 1]
```diff
@@ -2,5 +2,4 @@
 								// 1 will enable set background. 0 will disable set background.
 
 
-#define DOOR_CRUSH_DAMAGE 15
-#define USE_OPENSPACE
\ No newline at end of file
+#define DOOR_CRUSH_DAMAGE 15
```
Comment (fluorescent-net) [inline code/__defines/_compile_options.dm]: Please add new empty line at end of file.

File: code/__defines/misc.dm [hunk 2]
```diff
@@ -208,4 +208,7 @@
 #define ANNOUNSER_NAME "CEV Eris System Announcer"
 
 
-#define LIST_OF_CONSONANT list("b","c","d","f","g","h","j","k","l","m","n","p","q","r","s","t","v","w","x","y","z","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�")
\ No newline at end of file
+#define LIST_OF_CONSONANT list("b","c","d","f","g","h","j","k","l","m","n","p","q","r","s","t","v","w","x","y","z","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�","�")
+
+//Multi-z
+#define FALL_GIB_DAMAGE 999
```
Comment (fluorescent-net) [inline code/__defines/misc.dm]: Please add new empty line at end of file.

Comment (fluorescent-net) [inline code/__defines/misc.dm]: Please add new empty line at end of file.

**Pull request merged**